### PR TITLE
Locales properties restructuring

### PR DESCRIPTION
**Pull request by JuanJosePG**
First of all, delete all unused translations to do a better restructuring.

### CODE REVIEW

**Comment by JuanJosePG** [comment]
@Andresws12 check it the new structure for translations. If you agree with me, I'll modify all properties.

**Comment by Andresws12** [comment]
@JuanJosePG For me it's okay

File: src/localization/locales/locale.es.ts [hunk 1]
```diff
@@ -1,174 +1,78 @@
...
-    },
-    maintenance: {
-        title: 'Página en mantenimiento',
-        subtitle: 'Estaremos pronto de vuelta, gracias por su paciencia',
+        title: {
+            maintenance: 'Página en mantenimiento',
+            userAppointments: 'Mis Citas',
+            contact: 'Contacto',
+            register: 'Registro',
+            login: 'Iniciar sesion',
```
Comment (gonzalo-plaza) [inline src/localization/locales/locale.es.ts:64]: @JuanJosePG fix the word sesion, is sesión.

File: src/localization/locales/locale.es.ts [hunk 2]
```diff
@@ -1,174 +1,78 @@
...
-                usernameSame:
-                    'El nuevo nuevo nombre de usuario tiene que ser distinto al actual',
-                usernameChangeMessage:
-                    'El nombre de usuario ha sido actualizado',
+    common: {
+        email: 'E-Mail',
+        password: 'Contraseña',
+        components: {
+            appointments: {
+                pastAppointments: 'Citas pasadas',
```
Comment (gonzalo-plaza) [inline src/localization/locales/locale.es.ts:45]: @JuanJosePG Here "Citas pasadas" is good, but in my opinion I think would be better "Citas anteriores". Please @Andresws12 @PVillasana @pedrojgm Can you give our your opinion?

**Comment by gonzalo-plaza** [comment]
@JuanJosePG Is possible that this error will resolve when you fix the conflicts, but I take a screenshot with a problem in register form. components.loginInputs.inputEmpty

Sin título

File: src/localization/locales/locale.es.ts (hunk 1 above)
Comment (JuanJosePG) [reply src/localization/locales/locale.es.ts:64]: Fixed

File: src/localization/locales/locale.es.ts (hunk 2 above)
Comment (JuanJosePG) [reply src/localization/locales/locale.es.ts:45]: I'll wait to know the rest of the team's opinion to change it.

Comment (Andresws12) [reply src/localization/locales/locale.es.ts]: For me "Citas anteriores" is better

**Comment by Andresws12** [comment]
@gonzalo-plaza @JuanJosePG @PVillasana I fixed the conflics, please, review all code and layout

**Pull request merged**